aten/src/THCUNN/generic/MultiLabelMarginCriterion.cu

```diff
@@ -87,6 +87,7 @@ void THNN_(MultiLabelMarginCriterion_updateOutput)(
 
   THCTensor_(free)(state, input);
   THCIndexTensor_(free)(state, target);
+  THCTensor_(free)(state, istarget);
```

Comment:
Those modules must be used a lot haha. Maybe we should move their implementation to Python